### DISCUSSION TITLE
Removal of webpack 4

### DISCUSSION THREAD

**Discussion by heath-freenome** [storybookjs/storybook#21040] [General]
Many of the warnings NPM gives me about deprecated software in my system are coming from the webpack 4 embedded in `@storybook/core-common`. When will you remove embedded support for 4 and replace it with 5?

**Comment by bzh2610** [comment]
It appears that Storybook V7-beta works without webpack 4.

To list all of the dependencies requiring Webpack4 you can run `npm ls webpack`

Here's the output of my setup with Storybook V7 beta : 

It appears docs have not been updated just yet : https://storybook.js.org/docs/7.0/angular/builders/webpack

```
├─┬ @storybook/addon-postcss@3.0.0-alpha.1
│ ├─┬ css-loader@3.6.0
│ │ └── webpack@5.75.0 deduped
│ ├─┬ postcss-loader@4.3.0
│ │ └── webpack@5.75.0 deduped
│ └─┬ style-loader@1.3.0
│   └── webpack@5.75.0 deduped
├─┬ @storybook/angular@7.0.0-beta.35
│ ├─┬ @storybook/builder-webpack5@7.0.0-beta.35
│ │ ├─┬ babel-loader@9.1.2
│ │ │ └── webpack@5.75.0 deduped
│ │ ├─┬ fork-ts-checker-webpack-plugin@7.3.0
│ │ │ └── webpack@5.75.0 deduped
│ │ └── webpack@5.75.0 deduped
│ └── webpack@5.75.0 deduped
├─┬ @storybook/preset-scss@1.0.3
│ └─┬ style-loader@3.3.1
│   └── webpack@5.75.0 deduped
└─┬ babel-loader@8.2.5
  └── webpack@5.75.0 deduped
```

**Comment by WalterWeidner** [comment]
This is having the same impact on our project. `loader-utils` has a critical vulnerability and Storybook is the only dependency bringing that version into our project.

**Comment by alee8** [comment]
In a similar vein, when creating a repro issue related to use of fsevents@1, I noticed even the following repro creation cases (using `npx storybook repro`) below pulls in `@storybook/builder-webpack4@6.5.16` that has a dependency for `webpack@4.46.0`:
1. Select the repro framework (choose react), Select the repro base template (choose react)
2. Select the repro framework (choose react), Select the repro base template (choose webpack_react)

`webpack@4.46.0` always pulls in `chokidar@2.1.8` that then has a dependency for`fsevents@1.2.13`.

`fsevents@npm:1.2.13 is deprecated: fsevents 1 will break on node v14+ and could be using insecure binaries. Upgrade to fsevents 2.`

Would it be possible to have a repro framework/ repro base template that pulls in `@storybook/builder-webpack5`?
